src/FSharpLint.Console/Program.fs

```diff
@@ -79,6 +81,13 @@ let private start (arguments:ParseResults<ToolArgs>) (toolsPath:Ionide.ProjInfo.
         | Some _
         | None -> Output.StandardOutput() :> Output.IOutput
 
+    if arguments.Contains ToolArgs.Version then
+        let v = 
+            System.Reflection.Assembly.GetExecutingAssembly().GetCustomAttributes(false)
+            |> Seq.pick (fun a -> match a with | :? System.Reflection.AssemblyInformationalVersionAttribute as i -> Some i.InformationalVersion | _ -> None)
```

Comment:
@Thorium do you mind not using 1-char variable names? so rename v, a and i please (we in fact have a rule that detects this, we might enable it later)